content/blog/2025/05/camel412-whatsnew/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Apache Camel 4.12 What's New"
+date: 2025-05-28
+draft: false
+authors: [davsclaus]
+categories: ["Releases"]
+preview: "Details of what we have done in the Camel 4.12 release."
+---
+
+Apache Camel 4.12 has just been [released](/blog/2025/05/RELEASE-4.12.0/).
+
+This release introduces a set of new features and noticeable improvements that we will cover in this blog post.
+
+## Camel Core
+
+## Camel JBang
+
+## Camel Spring Boot
+
+The `camel-spring-boot` is upgraded to latest Spring Boot 3.5.0 release.
+
+## Miscellaneous
+
+Upgraded many third-party dependencies to the latest releases at the time of release.
+
+## New Components
+
+We have added a few new components:
+
+- `camel-dapr` - Dapr component which interfaces with Dapr Building Blocks.
+- `camel-pqc` - Post Quantum Computing Signature and Verification component.
```

Comment:
I think this should be Cryptography and not Computing, I will fix the description in the endpoint. Thanks.